src/DocumentFormat.OpenXml/Packaging/ImagePartTypeExtensions.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
```

Comment:
nit: there will be warnings when compiling with whitespace like this. We error out on CI builds if there are any warnings